static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-engagement-banner-row-support.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import { makeBannerABTestVariants } from 'common/modules/commercial/contributions-utilities';
+import {
+    getSupporterPaymentRegion as geolocationGetSupporterPaymentRegion,
+    getSync as geolocationGetSync,
+} from 'lib/geolocation';
+
+const componentType = 'ACQUISITIONS_ENGAGEMENT_BANNER';
+const abTestName = 'AcquisitionsEngagementBannerRowSupport';
+const INTsupportURL = 'https://support.theguardian.com/int';
+
+export const AcquisitionsEngagementBannerRowSupport: AcquisitionsABTest = {
+    id: abTestName,
+    campaignId: abTestName,
+    start: '2018-03-01',
+    expiry: '2018-05-24',
+    author: 'Santiago Villa Fernandez',
+    description:
+        'Points the "support the guardian" link in the engagement banner to the aud version of the support site',
```

Comment:
copy/paste hangover: int version, not aud version

Reply:
😅